cockpit/src/subscriptions-client.js

```diff
@@ -121,9 +122,27 @@ function getSubscriptionDetails() {
     });
 }
 
+/* convenience function for specifying d-bus strings */
+function s(value) {
```

Comment:
I like this refactoring 👍. Can we rename it to something like `dbus_str()`?

Reply:
Done